runtime/calamari/tests/integration_tests.rs

```diff
@@ -0,0 +1,950 @@
+		assert_eq!(Balances::usable_balance(&bob), 0);
+
+		let schedule = calamari_vesting::Pallet::<Runtime>::vesting_schedule();
+		let mut vested = 0;
+
+		for period in 0..schedule.len() {
+			// Timestamp expects milliseconds, so multiply by 1_000 to convert from seconds.
+			let now = schedule[period].1 * 1_000 + 1;
+			Timestamp::set_timestamp(now);
+			assert_ok!(CalamariVesting::vest(Origin::signed(bob.clone())));
+			vested += schedule[period].0 * unvested;
+			assert_eq!(Balances::usable_balance(&bob), vested);
+		}
+	});
+}
+
+#[test]
+fn verify_pallet_prefixes() {
+	fn is_pallet_prefix<P: 'static>(name: &str) {
+		// Compares the unhashed pallet prefix in the `StorageInstance` implementation by every
+		// storage item in the pallet P. This pallet prefix is used in conjunction with the
+		// item name to get the unique storage key: hash(PalletPrefix) + hash(StorageName)
+		// https://github.com/paritytech/substrate/blob/master/frame/support/procedural/src/pallet/
+		// expand/storage.rs#L389-L401
+		assert_eq!(
+			<calamari_runtime::Runtime as frame_system::Config>::PalletInfo::name::<P>(),
+			Some(name)
+		);
+	}
+
+	is_pallet_prefix::<calamari_runtime::System>("System");
+	is_pallet_prefix::<calamari_runtime::ParachainSystem>("ParachainSystem");
+	is_pallet_prefix::<calamari_runtime::Timestamp>("Timestamp");
+	is_pallet_prefix::<calamari_runtime::ParachainInfo>("ParachainInfo");
+	is_pallet_prefix::<calamari_runtime::TransactionPause>("TransactionPause");
+	is_pallet_prefix::<calamari_runtime::Balances>("Balances");
+	is_pallet_prefix::<calamari_runtime::TransactionPayment>("TransactionPayment");
+	is_pallet_prefix::<calamari_runtime::Democracy>("Democracy");
+	is_pallet_prefix::<calamari_runtime::Council>("Council");
+	is_pallet_prefix::<calamari_runtime::CouncilMembership>("CouncilMembership");
+	is_pallet_prefix::<calamari_runtime::TechnicalCommittee>("TechnicalCommittee");
+	is_pallet_prefix::<calamari_runtime::TechnicalMembership>("TechnicalMembership");
+	is_pallet_prefix::<calamari_runtime::Authorship>("Authorship");
+	is_pallet_prefix::<calamari_runtime::CollatorSelection>("CollatorSelection");
+	is_pallet_prefix::<calamari_runtime::Session>("Session");
+	is_pallet_prefix::<calamari_runtime::Aura>("Aura");
+	is_pallet_prefix::<calamari_runtime::AuraExt>("AuraExt");
+	is_pallet_prefix::<calamari_runtime::Treasury>("Treasury");
+	is_pallet_prefix::<calamari_runtime::Scheduler>("Scheduler");
+	is_pallet_prefix::<calamari_runtime::XcmpQueue>("XcmpQueue");
+	is_pallet_prefix::<calamari_runtime::PolkadotXcm>("PolkadotXcm");
+	is_pallet_prefix::<calamari_runtime::CumulusXcm>("CumulusXcm");
+	is_pallet_prefix::<calamari_runtime::DmpQueue>("DmpQueue");
+	is_pallet_prefix::<calamari_runtime::Utility>("Utility");
+	is_pallet_prefix::<calamari_runtime::Multisig>("Multisig");
+	is_pallet_prefix::<calamari_runtime::Sudo>("Sudo");
+	is_pallet_prefix::<calamari_runtime::CalamariVesting>("CalamariVesting");
```

Comment:
@stechu actually I'm not clear what your suggestion here was.

Reply:
i think the idea was to wrap each individual line into a rust macro, so that the line reads simply e.g.
> pallet_storage_prefix_is!("Multisig")
abstracting away the verbosity of the rust syntax.
One param should be enough for now as the names and identifiers all seem to be identical

Reply:
Not sure how to reduce the verbosity further. I gotta pass a generic for the `name` function. And I gotta pass the string to compare it to in the assertion.

Reply:
Ideally the macro takes a list of module name and generate this code automatically, we can leave this to future PRs.

Reply:
So for example if I pass it a string token `AuraExt` the macro can create this generic `calamari_runtime::AuraExt>`